From f814c027ee15365d34dd026430ddde5c284641f2 Mon Sep 17 00:00:00 2001
From: Russell King <rmk+kernel@arm.linux.org.uk>
Date: Fri, 15 May 2015 19:22:45 +0100
Subject: [PATCH 090/148] staging: etnaviv: clean up public API (part 2)

This fixes the preceding patch for updates I have in my tree.

Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 drivers/staging/etnaviv/etnaviv_gem_submit.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/staging/etnaviv/etnaviv_gem_submit.c b/drivers/staging/etnaviv/etnaviv_gem_submit.c
index 301983a..746cb55 100644
--- a/drivers/staging/etnaviv/etnaviv_gem_submit.c
+++ b/drivers/staging/etnaviv/etnaviv_gem_submit.c
@@ -259,13 +259,6 @@ static int submit_reloc(struct etnaviv_gem_submit *submit, struct etnaviv_gem_ob
 			return -EINVAL;
 		}
 
-		/* Vivante hardware has no need for shifts or bitwise or-ing */
-		if (submit_reloc.shift || submit_reloc.or) {
-			DRM_ERROR("invalid shift or bitwise or at reloc %u\n",
-				  i);
-			return -EINVAL;
-		}
-
 		ret = submit_bo(submit, submit_reloc.reloc_idx, &bobj, &iova);
 		if (ret)
 			return ret;
-- 
2.6.1

